src/gdb_packet.c

```diff
@@ -332,13 +342,15 @@ void gdb_packet_send(const gdb_packet_s *const packet)
 
 void gdb_put_packet(const char *preamble, size_t preamble_size, const char *data, size_t data_size, bool hex_data)
 {
+	gdb_packet_s *packet_buffer = gdb_full_packet_buffer();
```

Comment:
Same comment about the local here as above (and in the functions below)

Reply:
Also done